### PR TITLE
*: enable gci

### PR DESCRIPTION
**Pull request by hawkingrei**
### What problem does this PR solve?

Issue Number: ref #35345

Problem Summary:

### What is changed and how it works?

use ```gci``` to replace with ```goimports``` because ```goimports``` have many problems in the nogo
### Check List

Tests 

- [ ] Unit test
- [ ] Integration test
- [x] Manual test (add detailed scripts or steps below)
- [ ] No code

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
None
```

### CODE REVIEW

**Review by ngaut — APPROVED** [review]

**Comment by hawkingrei** [comment]
The order of go import in the tidb is so chaos.

**Review by leiysky — APPROVED** [review]

File: br/pkg/lightning/backend/kv/sql2kv.go [hunk 1]
```diff
@@ -33,7 +33,9 @@ import (
 	"github.com/pingcap/tidb/expression"
 	"github.com/pingcap/tidb/meta/autoid"
 	"github.com/pingcap/tidb/parser/model"
-	"github.com/pingcap/tidb/parser/mysql"
+	"github.com/pingcap/tidb/parser/mysql" //nolint:goimports
+	// Import tidb/planner/core to initialize expression.RewriteAstExpr
+	_ "github.com/pingcap/tidb/planner/core"
```
Comment (tisonkun) [inline br/pkg/lightning/backend/kv/sql2kv.go:38]: What if separate init only import-s?

**Review by tisonkun — COMMENTED** [review]
Generally looks good. One suggestion inline. Also you may remember to update code style section on TiDB Dev Guide https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/code-style-and-quality-guide.html.

File: br/pkg/lightning/backend/kv/sql2kv.go (hunk 1 above)
Comment (hawkingrei) [reply br/pkg/lightning/backend/kv/sql2kv.go:38]: Let me try it again.

Comment (hawkingrei) [reply br/pkg/lightning/backend/kv/sql2kv.go:38]:
```
[2022-07-03T15:57:37.755Z] br/pkg/lightning/backend/kv/sql2kv.go:36: File is not `goimports`-ed (goimports)

[2022-07-03T15:57:37.755Z] 	"github.com/pingcap/tidb/parser/mysql"

```

Comment (hawkingrei) [reply br/pkg/lightning/backend/kv/sql2kv.go:38]: if remove ```//nolint:goimports```

```
[2022-07-03T15:57:37.755Z] br/pkg/lightning/backend/kv/sql2kv.go:36: File is not `goimports`-ed (goimports)

[2022-07-03T15:57:37.755Z] 	"github.com/pingcap/tidb/parser/mysql"

```

I don't know what is it.

**Review by tangenta — APPROVED** [review]

**Review by xiongjiwei — APPROVED** [review]

**Pull request merged**